Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/TurnPointerOnOffExample/TurnPointersOnOff.cs

```diff
@@ -0,0 +1,134 @@
+        {
+            PointerUtils.SetGrabPointerBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off, Handedness.Any);
+        }
+
+        public void SetPokeEnabled(bool isEnabled)
+        {
+            PointerUtils.SetPokePointerBehavior(isEnabled ? PointerBehavior.Default : PointerBehavior.Off, Handedness.Any);
+        }
+
+        public void SetVR()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        public void SetFingerOnly()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        public void SetHoloLens1()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Off, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Default);
+        }
+
+        public void SetHoloLens2()
+        {
+            PointerUtils.SetPokePointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGrabPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetRayPointerBehavior(PointerBehavior.Default, Handedness.Any);
+            PointerUtils.SetGGVBehavior(PointerBehavior.Off);
+        }
+
+        void Start()
+        {
+            IMixedRealityCapabilityCheck capabilityChecker = CoreServices.InputSystem as IMixedRealityCapabilityCheck;
+            if (capabilityChecker != null)
+            {
+                if (capabilityChecker.CheckCapability(MixedRealityCapability.ArticulatedHand))
+                {
+                    SetHoloLens2();
+                }
+                else if (capabilityChecker.CheckCapability(MixedRealityCapability.MotionController))
+                {
+                    SetVR();
+                }
+                else
+                {
+                    SetHoloLens1();
+                }
+            }
+            else
+            {
+                Debug.LogWarning("Input system does not implement IMixedRealityCapabilityCheck, not setting to any preset interaction");
+            }
+
+            PopulatePointerToggleList();
+        }
+
+        private void PopulatePointerToggleList()
+        {
+            foreach (var icl in FindObjectsOfType<Interactable>())
+            {
+                if (icl.Dimensions == 2)
```

Comment:
Nit: Add a comment on the significance of 2

Reply:
I instead added a property IsToggle on interactable